guides/common/modules/proc_configuring-server-to-consume-content-from-a-custom-cdn.adoc

```diff
@@ -0,0 +1,39 @@
+[id="configuring-{project-context}-server-to-consume-content-from-a-custom-cdn_{context}"]
+= Configuring {ProjectServerTitle} to Consume Content from a Custom CDN
+
+If you have an internal Content Delivery Network (CDN) server, you can configure your {ProjectServer} to consume Red{nbsp}Hat repositories from this CDN server instead of the Red{nbsp}Hat CDN.
+A CDN server can be any web server that mirrors repositories in the same directory structure as the Red{nbsp}Hat CDN.
+
+You can configure the source of content for each organization.
+{Project} recognizes automatically which Red{nbsp}Hat repositories from the subscription manifest in your organization are available on your CDN server.
+
+.Prerequisite
+* You have a CDN server that provides Red{nbsp}Hat content and is accessible by {ProjectServer}.
+* If your CDN server uses HTTPS, ensure you have uploaded the SSL certificate into {Project}.
+For more information, see {ContentManagementDocURL}Importing_Custom_SSL_Certificates_content-management[Importing Custom SSL Certificates] in _{ContentManagementDocTitle}_.
+* You have uploaded a manifest to your organization.
+
+.Procedure
+. In the {ProjectWebUI}, navigate to *Content* > *Subscriptions*.
+. Click the *Manage Manifest* button.
+. Select the *CDN Configuration* tab.
+. Select the *Custom CDN* tab.
+. In the *URL* field, enter the URL of your CDN server from which you want {ProjectServer} to consume Red{nbsp}Hat repositories.
+. Optional: In the *SSL CA Content Credential*, select the SSL certificate of the CDN server.
```

Comment:
You might want to mention that the CA Certificate needs to be added to the sat as a content credential (this dropdown only shows content credentials.) May be that is obvious. Leave it to you :)

Reply:
Wait its in the prereq. N/M

Reply:
@parthaa That's it in Prerequisites :)